pkg/healthcheck/healthcheck.go

```diff
@@ -932,3 +960,39 @@ func validateDataPlanePodReporting(pods []*pb.Pod) error {
 
 	return nil
 }
+
+func checkUnschedulablePods(pods []corev1.Pod) error {
+	var podName []string
+	for _, pod := range pods {
+		for _, condition := range pod.Status.Conditions {
+			if condition.Reason == "Unschedulable" {
+				podName = append(podName, pod.Name)
+			}
+		}
+	}
+
+	if len(podName) != 0 {
+		pods := strings.Join(podName, ",")
+		return fmt.Errorf("No node available for the pods %s", pods)
```

Comment:
Can we change this to `Unable to schedule pods: %s`?

Reply:
sure, meanwhile i created an issue[https://github.com/linkerd/website/issues/272](url) on the linkerd website, will make a pr soon, just needed some help in what solutions should i write.